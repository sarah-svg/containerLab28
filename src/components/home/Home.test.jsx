import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Home';

describe('Header component', () => {
  afterEach(() => cleanup());
  it('renders Header', () => {
    const { asFragment } = render(<Header handleSearch={jest.fn()} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
