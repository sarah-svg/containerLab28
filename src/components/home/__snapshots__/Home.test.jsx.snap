// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header 1`] = `
<DocumentFragment>
  <header
    style="height: 120px; border: 1px solid black;"
  >
    <h1
      style="text-align: center;"
    >
      News Search
    </h1>
    <form
      style="text-align: center;"
    >
      <input
        data-testid="search-input"
        placeholder="Search all articles..."
      />
    </form>
  </header>
</DocumentFragment>
`;
